main.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,900');
+@import url('https://fonts.googleapis.com/css?family=Raleway:400,600');
+
+html {
+  font-size: 16px;
+}
+body {
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
+}
+
+
+
+/* Programme Builder CELLS */
+
+.prog-cell {
```

Comment:
We need to make sure that this class (or perhaps a more _semantically accurate one_) is **only applied** to the scheduled sessions. At the moment every cell is given a border which is not the desired effect from the wireframes.